src/core_modules/capture-ui/PreviewImage/PreviewImage.component.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = () => ({
+    container: {
+        position: 'relative',
+        display: 'inline-block',
+        '&:hover $image': {
+            opacity: 0.5,
+        },
+        '&:hover $icon': {
+            visibility: 'visible',
+        },
+    },
+    image: {
+        height: 80,
```

Comment:
To make sure it doesn't exceed the maximum size, can you use `maxHeight` instead? Thanks!

Reply:
We get a fixed height for the image by using `height`, so 80 pixels won't be exceeded. On the other hand if we switch to `maxHeight` we might end up with an image lower than 80 pixels.

Reply:
Talked to @superskip, he will make sure the image-container is always 120x80 (other elements should not move around when the image is loaded) etc.